triggers-examples/functions/sendOrderConfirmationText.js

```diff
@@ -0,0 +1,15 @@
+async function sendOrderConfirmationText({ fullDocument }) {
```

Comment:
Can we add another very simple function like this demonstrating a very basic axios request to a[ public api](https://apipheny.io/free-api/)? something from here? e.g. https://datausa.io/api/data?drilldowns=Nation&measures=Population

Reply:
what type of trigger would you want for that?

Reply:
you'd want this as a DB Trigger as well? 

why do you want this in addition to the twilio example?

Reply:
Twilio is a great example of using an npm package for a 3rd party, but the reason i want an HTTPS request is to demonstrate the range of how integrations can work even without npm support

Reply:
ah ok. maybe we can add that later if we do another set of examples for the data api or smthn.